backend/src/migrations/20240210124346_create_tables.ts

```diff
@@ -11,27 +11,8 @@ async function up(knex: Knex): Promise<void> {
     await knex.schema.createTable(DatabaseTableName.USERS, (table) => {
         table.uuid(DatabaseColumnName.ID).primary();
         table.string(DatabaseColumnName.EMAIL).unique().notNullable();
-        table.text(DatabaseColumnName.PASSWORD_HASH).notNullable();
-        table.text(DatabaseColumnName.PASSWORD_SALT).notNullable();
-        table
-            .dateTime(DatabaseColumnName.CREATED_AT)
-            .notNullable()
-            .defaultTo(knex.fn.now());
-        table
-            .dateTime(DatabaseColumnName.UPDATED_AT)
-            .notNullable()
-            .defaultTo(knex.fn.now());
-    });
-    await knex.schema.createTable(DatabaseTableName.OAUTH_USERS, (table) => {
-        table.uuid(DatabaseColumnName.ID).primary();
-        table.string(DatabaseColumnName.EMAIL).notNullable();
-        table
-            .enu(
-                DatabaseColumnName.OAUTH_STRATEGY,
-                Object.values(OauthStrategy),
-            )
-            .notNullable();
-        table.string(DatabaseColumnName.OAUTH_ID).notNullable();
+        table.text(DatabaseColumnName.PASSWORD_HASH);
```

Comment:
If files with previous migrations have already used on staging, you should consider to change DB structure by creating new migration file.